packages/genesys-spark-tokens/snapshot/scss/gse-ui-legacy-light.scss

```diff
@@ -691,7 +691,7 @@
   --gse-ui-toggle-focus-border-color: #aac9ff;
   --gse-ui-toggle-focus-border-width: 2px;
   --gse-ui-toggle-focus-border-style: solid;
-  --gse-ui-toggle-focus-borderRadius: 16px;
+  --gse-ui-toggle-focus-borderRadius: 4px;
```

Comment:
@VirtualClover is this an expected change? I thought dark mode would not affect the legacy theme.

Reply:
Ah yes, I added the fix for the toggle mentioned by Chris here: 
https://inindca.atlassian.net/browse/COMUI-3211?focusedCommentId=6074203